pimd/pim_cmd.c

```diff
@@ -6861,6 +6877,35 @@ DEFUN (no_ip_pim_packets,
 	return CMD_SUCCESS;
 }
 
+DEFPY (igmp_group_watermark,
+       igmp_group_watermark_cmd,
+       "ip igmp watermark-warn (10-60000)$limit",
+       IP_STR
+       IGMP_STR
+       "Configure group limit for watermark warning\n"
+       "Group count to generate watermark warning\n")
+{
+	PIM_DECLVAR_CONTEXT(vrf, pim);
+	pim->igmp_watermark_limit = limit;
+
+	return CMD_SUCCESS;
+}
+
+DEFPY (no_igmp_group_watermark,
+       no_igmp_group_watermark_cmd,
+       "no ip igmp watermark-warn [(10-60000)$limit]",
+       NO_STR
+       IP_STR
+       IGMP_STR
+       "Unconfigure watermark warning limit for igmp groups\n"
+       "Group count\n")
```

Comment:
can we use the same CLI descriptions? `Group count to generate watermark warning`

Reply:
@ton31337 Done.